src/visualization/plot_recipes.jl

```diff
@@ -188,6 +196,23 @@ returns a `DiffEqBase.ODESolution`) or Trixi's own `solve!` (which returns a
 """
 PlotData2D(sol::TrixiODESolution; kwargs...) = PlotData2D(sol.u[end], sol.prob.p; kwargs...)
 
+function _get_orientations(mesh, slice_axis)
+  if ndims(mesh) == 2 || (ndims(mesh) == 3 && slice_axis === :z)
+    orientation_x = 1
+    orientation_y = 2
+  elseif ndims(mesh) == 3 && slice_axis === :y
+    orientation_x = 1
+    orientation_y = 3
+  elseif ndims(mesh) == 3 && slice_axis === :x
+    orientation_x = 2
+    orientation_y = 3
+  else
+    orientation_x = 0
+    orientation_y = 0
+  end
+  return orientation_x, orientation_y
+end
```

Comment:
Could you please check the ordering of the axes in 3D, @sloede?

Reply:
@erik-f You implemented the conversion 3D-2D in Trixi2Img, didn't you? Could you please check whether the ordering of the 2D axes from 3D results is correct? I took it from https://github.com/trixi-framework/Trixi.jl/blob/44fefb054c43e37c00c1468edb3d99eb5b7826fa/src/visualization/interpolate.jl#L65-L76

Reply:
Looks alright to me, and the examples I tested look fine too.

Reply:
Thanks, @erik-f :+1: